.github/workflows/ci.yml

```diff
@@ -0,0 +1,64 @@
+name: CI
+
+on:
+  # Triggers the workflow on push or pull request events but only for the "develop" branch
+  push:
+    branches: [ "develop" ]
+  pull_request:
+    branches: [ "develop" ]
+
+  workflow_dispatch:
+
+jobs:
+  build-run-tests:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+    container: ghcr.io/ggeorgakoudis/ams-ci-test-ruby-centos7
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - uses: actions/checkout@v3
+      - name: Build AMS
+        shell: bash -l {0}
+        run: |
+          module load gcc/11.2.1
+          source /spack/share/spack/setup-env.sh
+          spack env activate -p /ams-spack-env
+          mkdir build
+          cd build
+          export AMS_MFEM_PATH=`spack location -i mfem`
+          export AMS_TORCH_PATH=$(spack location -i py-torch)/lib/python3.10/site-packages/torch/share/cmake/Torch
+          export AMS_FAISS_PATH=`spack location -i faiss`
+          export AMS_UMPIRE_PATH=`spack location -i umpire`
+          export AMS_HIREDIS_PATH=`spack location -i hiredis`
+          export AMS_REDIS_PLUS_PLUS_PATH=`spack location -i redis-plus-plus`
```

Comment:
@koparasy This is redundant too, remove?

Reply:
Same as above.

Reply:
superseded by above comment